File: talk.py
#!/Users/ykiyoshima/opt/anaconda3/envs/satominband/bin/python
# -*- coding: utf-8 -*-

import sys
import requests
import os
import cgi
from dotenv import load_dotenv
load_dotenv()

storage = cgi.FieldStorage()
data = storage.getvalue('data')
apikey = 'DZZNx6Z5eBHH7qFixKooNc4XRLIZ4Q4V'
talk_url = "https://api.a3rt.recruit.co.jp/talk/v1/smalltalk"
payload = {"apikey": apikey, "query": data}
response = requests.post(talk_url, data=payload)
try:
    print(response.json()["results"][0]["reply"])
except:
    print(response.json())
    print("ごめんなさい。もう一度教えて下さい。")

File: talk.php
<?php
// 自動で読み込み
require './vendor/autoload.php';
// .envを使用する
Dotenv\Dotenv::createImmutable(__DIR__)->load();

$transcript = $_GET['transcript'];
// $encode_transcript = urlencode($transcript);
$url = "https://api.a3rt.recruit.co.jp/talk/v1/smalltalk";

$palam=array(
    'query' => $transcript,
    'apikey' => $_ENV["CHAT_API_KEY"],
);

$ch = curl_init();
$headr = array();
$headr[] = 'Content-Type: application/json;';
curl_setopt($ch, CURLOPT_URL, $url);
curl_setopt($ch, CURLOPT_POST, true);
curl_setopt($ch, CURLOPT_FOLLOWLOCATION, true);
curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
curl_setopt($ch, CURLOPT_SSL_VERIFYHOST, 0);
curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, false);
curl_setopt($ch, CURLOPT_POSTFIELDS, http_build_query($palam));

//API実行
$result = curl_exec($ch);

curl_close($ch);

$rec = json_decode($result,true);
echo $rec['results'][0]['reply'];